sidebars.js

```diff
@@ -24,10 +24,10 @@ module.exports = {
             "aprender-eosio/desarrollo-de-dapps",
             "aprender-eosio/cadena-de-bloques-eos",
             "aprender-eosio/contratos-inteligentes",
-            "aprender-eosio/contratos-ricardianos"
+            "aprender-eosio/contratos-ricardianos",
+            "aprender-eosio/componentes-eosio"
         ],
         "Tutoriales": [
-            "tutoriales/como-iniciar",
```

Comment:
Is this remove right?

Reply:
yes we consolidated this into a new link for eosio-components